src/server/mod.rs

```diff
@@ -130,6 +130,10 @@ macro_rules! build_app {
             .service(
                 web::resource("/1.0/sync/1.5".to_string()).route(web::get().to(tokenserver::get)),
             )
+            .service(
```

Comment:
Is this to route numeric non-uuid user ids to get? I really don't understand why the functional tests passed before this.

Reply:
Yeah, I'm not convinced we need it, but I was getting 404s trying to pass anything to that URL and wasn't sure that it was required. I think the larger problem is that the tests are using the wrong path.